src/CMakeLists.txt

```diff
@@ -1,4 +1,6 @@
 set_source_files_properties(${PROJECT_SOURCE_DIR}/third_party/jsoncpp/jsoncpp.cpp PROPERTIES COMPILE_FLAGS -w)
+set_source_files_properties(${PROJECT_SOURCE_DIR}/src/p11engine.cc PROPERTIES COMPILE_FLAGS -w)
```

Comment:
Dare I ask what necessitated hiding warnings from that file?

Reply:
The problem here is that one api call is marked as deprecated but they have not added not deprecated call for keypair generation

Reply:
Ok, maybe it is better to hide only deprecated warnings. I will update it